File: discover_ruby/write_them_to_disc.rb
require 'httpclient'
require 'json'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 89a061b4ed9ecbfd9801450418d3af551b06c4ae'
}

filename = '/tmp/33'
uri = "https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc"
target = open(filename,'w')
target.truncate(0)
clin = HTTPClient.new

res = clnt.get_content uri, extheaders
target.write(res)
puts "The file was saved!"
target.close

File: discover_more_python/write_them_to_disc.py
import requests

headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token 89a061b4ed9ecbfd9801450418d3af551b06c4ae'
}  
url = 'https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc'
r = requests.get(url, headers=headers)
target = open ("/tmp/33.txt", 'w')
target.write(r.content)
target.close()
print "The file was saved!"
